repl/src/test/scala/com/cloudera/livy/repl/PythonSessionSpec.scala

```diff
@@ -167,5 +168,34 @@ class PythonSessionSpec extends BaseSessionSpec {
 
     result should equal (expectedResult)
   }
+}
+
+class Python2SessionSpec extends PythonSessionSpec {
+  override def createInterpreter(): Interpreter = PythonInterpreter(new SparkConf(), PySpark())
+}
+
+class Python3SessionSpec extends PythonSessionSpec {
+
+  override protected def withFixture(test: NoArgTest): Outcome = {
+    assume(!sys.props.getOrElse("skipPySpark3Tests", "false").toBoolean, "Skipping PySpark3 tests.")
+    test()
+  }
+
+  override def createInterpreter(): Interpreter = PythonInterpreter(new SparkConf(), PySpark3())
 
+  it should "execute `1 / 2` == 0.5" in withSession { session =>
```

Comment:
Is this just to check it's really running python 3? Wouldn't it be easier to just check `sys.version`?